src/main/java/spireMapOverhaul/zones/brokenSpace/relics/BrokenMatryoshka.java

```diff
@@ -0,0 +1,62 @@
+package spireMapOverhaul.zones.brokenSpace.relics;
+
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.relics.AbstractRelic;
+import com.megacrit.cardcrawl.relics.Matryoshka;
+import com.megacrit.cardcrawl.rewards.RewardItem;
+
+public class BrokenMatryoshka extends BrokenRelic {
+    public static final String ID = "BrokenMatryoshka";
+    public static final int AMOUNT = 4;
+    public static final int EMPTY = 2;
+
+    public BrokenMatryoshka() {
+        super(ID, RelicTier.SPECIAL, LandingSound.CLINK, Matryoshka.ID);
+    }
+
+    private RelicTier getRelicTier() {
+        return AbstractDungeon.relicRng.randomBoolean(0.25F) ? RelicTier.COMMON : RelicTier.UNCOMMON;
+    }
+
+    @Override
+    public void onEquip() {
+        for (int i = 0; i < AMOUNT-1; i++) {
+            AbstractRelic r = AbstractDungeon.returnRandomScreenlessRelic(getRelicTier());
+            AbstractDungeon.getCurrRoom().rewards.add(new RewardItem(r));
+        }
+        AbstractRelic r = AbstractDungeon.returnRandomScreenlessRelic(getRelicTier());
+        RewardItem reward = new RewardItem(r);
+        AbstractDungeon.getCurrRoom().rewards.add(reward);
+        if (!Settings.hasSapphireKey){
```

Comment:
This should also check `Settings.isFinalActAvailable` (you can see that `AbstractChest.open` checks that when it calls this same method).